### PR TITLE
Readme file

### PR DESCRIPTION
**Pull request by YuliiaOsadchuk**

### CODE REVIEW

**Review by bohdan-shtohrinets-sigma — CHANGES_REQUESTED** [review]
-

File: src/setupTests.ts [hunk 1]
```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
```
Comment (bohdan-shtohrinets-sigma) [inline src/setupTests.ts]: pls revert this file deletion

File: src/reportWebVitals.ts [hunk 2]
```diff
@@ -1,15 +0,0 @@
-import { ReportHandler } from 'web-vitals';
-
-const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```
Comment (bohdan-shtohrinets-sigma) [inline src/reportWebVitals.ts]: pls revert this file deletion

**Review by bohdan-shtohrinets-sigma — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -2,24 +2,58 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+# Directory structure
+
+```
+task-manager-app/
+   README.md
+   node_modules/
```
Comment (bohdan-shtohrinets-sigma) [inline README.md]: node_modules are not necessary in project structure doc. Only include files related to development: `src/`,

**Review by bohdan-shtohrinets-sigma — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -2,24 +2,53 @@
...
+        signInPage
+          SignInPage.tsx
+          SignInPage.styles.ts
+      store
+        store.ts
+        slices.ts
+        selector.ts
+```
+
 ## Available Scripts
```
Comment (b-sht-lionwood) [inline README.md]: this is unussed code

File: README.md [hunk 5]
```diff
@@ -2,24 +2,53 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (b-sht-lionwood) [inline README.md:3]: create simple guide of how to run the application

**Review by b-sht-lionwood — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -2,24 +2,53 @@
...
+# Directory structure
+
+```
+task-manager-app/
+	tsconfig.json
+    src/
+      app.tsx
+      index.tsx
+      interfaces.ts
+      enum.ts
```
Comment (b-sht-lionwood) [inline README.md]: enums

File: README.md [hunk 7]
```diff
@@ -2,24 +2,53 @@
...
+      constants.ts
+      global.styles.ts
+      server/
+        db.json
+      components/
+        signInForm
```
Comment (b-sht-lionwood) [inline README.md]: create naming for files in following structure:
sign-in-form/
sign-in-form.tsx

File: README.md [hunk 8]
```diff
@@ -2,24 +2,53 @@
...
+        homePage
+          HomePage.tsx
+          HomePage.styles.ts
+        signInPage
+          SignInPage.tsx
+          SignInPage.styles.ts
+      store
+        store.ts
+        slices.ts
+        selector.ts
```
Comment (b-sht-lionwood) [inline README.md]: selectors

**Review by b-sht-lionwood — CHANGES_REQUESTED** [review]

File: README.md [hunk 9]
```diff
@@ -1,46 +1,111 @@
-# Getting Started with Create React App
+# Getting Started
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+To launch the app run `npm i`.
```
Comment (b-sht-lionwood) [inline README.md]: to install dependencies run 
```
npm install
```

File: README.md [hunk 10]
```diff
@@ -1,46 +1,111 @@
...
 
-In the project directory, you can run:
-
-### `yarn start`
-
-Runs the app in the development mode.\
+To run the app in the development mode run `npm run start`
```
Comment (b-sht-lionwood) [inline README.md]: same here

**Review by b-sht-lionwood — APPROVED** [review]

**Pull request merged**